CondTools/L1TriggerExt/interface/L1ConfigOnlineProdBaseExt.h

```diff
@@ -41,10 +40,13 @@ class L1ConfigOnlineProdBaseExt : public edm::ESProducer {
 
 private:
   // ----------member data ---------------------------
+  edm::ESGetToken<L1TriggerKeyListExt, L1TriggerKeyListExtRcd> keyList_token;
+  edm::ESGetToken<L1TriggerKeyExt, L1TriggerKeyExtRcd> key_token;
 
 protected:
   l1t::OMDSReader m_omdsReader;
   bool m_forceGeneration;
+  edm::ESConsumesCollectorT<TRcd> m_setWhatProduced(const edm::ParameterSet&);
```

Comment:
I think this function has a confusing name, typicalle `m_` prefix has been used for member variables.

Reply:
True, will update to a better name